bin/dune_cmd.ml

```diff
@@ -0,0 +1,88 @@
+open! Stdune
+open Import
+
+let all : _ Cmdliner.Cmd.t list =
```

Comment:
Is it necessary to move code around? We have some internal patches (like adding new subcommands) that will require some extra work. If we can avoid moving the code that would be helpful.

Reply:
Unfortunately this is the only way I could think of. The entry point for Dune is the same as the cmdliner definition which means the cmdliner cmd cannot be shared with other executables without placing it in its own file.

Reply:
I see, thanks. This is certainly not a blocker, just an extra cost of the current implementation to keep in mind (perhaps, when comparing with alternatives).